app/src/main/kotlin/com/wire/android/ui/sharing/ImportMediaScreen.kt

```diff
@@ -0,0 +1,207 @@
+                            itemSpacing = dimensions().spacing8x
+                        ) { page ->
+                            ImportedMediaItemView(importedItemsList[page], isMultipleImport, importMediaViewModel.wireSessionImageLoader)
+                        }
+                    }
+                }
+                Divider(
+                    modifier = Modifier.padding(vertical = dimensions().spacing16x),
+                    color = MaterialTheme.wireColorScheme.divider,
+                    thickness = Dp.Hairline
+                )
+                SearchTopBar(
+                    isSearchActive = searchBarState.isSearchActive,
+                    searchBarHint = stringResource(
+                        R.string.search_bar_conversations_hint,
+                        stringResource(id = R.string.conversations_screen_title).lowercase()
+                    ),
+                    searchQuery = searchBarState.searchQuery,
+                    onSearchQueryChanged = {
+                        importMediaViewModel.onSearchQueryChanged(it)
+                        searchBarState.searchQueryChanged(it)
+                    },
+                    onInputClicked = searchBarState::openSearch,
+                    onCloseSearchClicked = searchBarState::closeSearch
+                )
+                ConversationList(
+                    modifier = Modifier.weight(1f),
+                    lazyListState = lazyListState,
+                    conversationListItems = persistentMapOf(
+                        ConversationFolder.Predefined.Conversations to importMediaViewModel.shareableConversationListState.searchResult
+                    ),
+                    conversationsAddedToGroup = importMediaViewModel.shareableConversationListState.conversationsAddedToGroup,
+                    isSelectableList = true,
+                    onConversationSelectedOnRadioGroup = importMediaViewModel::selectConversationOnRadioGroup,
+                    searchQuery = searchBarState.searchQuery.text,
+                    onOpenConversation = importMediaViewModel::onConversationClicked,
+                    onEditConversation = {},
+                    onOpenUserProfile = {},
+                    onOpenConversationNotificationsSettings = {},
+                    onJoinCall = {}
+                )
+            }
+            BackHandler(enabled = searchBarState.isSearchActive) {
+                searchBarState.closeSearch()
+            }
+        },
+        bottomBar = {
+            SendContentButton(
+                mainButtonText = actionButtonTitle,
+                count = if (importMediaViewModel.importMediaState.importedAssets.size > 0) {
+                    importMediaViewModel.shareableConversationListState.conversationsAddedToGroup.size
+                } else {
+                    0
+                },
+                onMainButtonClick = importMediaViewModel::onImportedMediaSent
```

Comment:
I think more appropiate name would be something like sendImportedMedia ? Dunno onImportMediaSent is suggesting that ImportMedia has already happend

Reply:
renamed